__tests__/Unit/Components/Standup/StandupContainer.test.tsx

```diff
@@ -0,0 +1,227 @@
+import StandUpContainer from '@/components/standup';
+import { renderWithRouter } from '@/test_utils/createMockRouter';
+import { Provider } from 'react-redux';
+import { store } from '@/app/store';
+import { fireEvent, screen, waitFor } from '@testing-library/react';
+import { setupServer } from 'msw/node';
+import handlers from '../../../../__mocks__/handlers';
+import {
+    ERROR_MESSAGE,
+    STANDUP_SUBMISSION_SUCCESS,
+} from '@/constants/constants';
+import { failedPostStandup } from '../../../../__mocks__/handlers/standup.handler';
+import { ToastContainer } from 'react-toastify';
+import * as SaveProgressHook from '@/app/services/progressesApi';
+import { throws } from 'assert';
+
+const server = setupServer(...handlers);
+
+describe('StandupContainer', () => {
+    beforeAll(() => {
+        server.listen();
+    });
+    afterEach(() => {
+        server.resetHandlers();
+    });
+    afterAll(() => {
+        server.close();
+    });
+    test('should render completed inputField ', function () {
+        renderWithRouter(
+            <Provider store={store()}>
+                <StandUpContainer />
+            </Provider>
+        );
+        const input = screen.getByTestId(
+            'completedInputField'
+        ) as HTMLInputElement;
```

Comment:
we should prefer get by role and use label rather than get by id.

Reply:
fixed